docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+
+   ```yaml
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: domain1-admin-server-external-lb
+     namespace: default
+   spec:
+     ports:
+     - name: default
+       port: 7001
+       protocol: TCP
+       targetPort: 7001
+     selector:
+       weblogic.domainUID: domain1
+       weblogic.serverName: admin-server
+     sessionAffinity: None
+     type: LoadBalancer
+   ```
+
+   Use the sample configuration file `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/domain-on-pv/cluster-lb.yaml` to create a load balancer service for the managed servers. If you are choosing not to use the predefined YAML file and instead created new one with customized values, then substitute the following content with you domain values.
+
+   ```yaml
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: domain1-cluster-1-lb
+     namespace: default
+   spec:
+     ports:
+     - name: default
+       port: 8001
+       protocol: TCP
+       targetPort: 8001
+     selector:
+       weblogic.domainUID: domain1
+       weblogic.clusterName: cluster-1
+     sessionAffinity: None
+     type: LoadBalancer
+   ```
+
+   Create the load balancer services using the following commands:
+
+   ```bash
+   $ kubectl apply -f ~/azure/weblogic-on-aks/admin-lb.yaml
+   service/domain1-admin-server-external-lb created
+
+   $ kubectl  apply -f ~/azure/weblogic-on-aks/cluster-lb.yaml
+   service/domain1-cluster-1-external-lb created
+   ```
+
+   After a short time, you will see the Administration Server and Managed Servers running.
+
+   Use the following command to check server pod status:
+
+   ```bash
+   $ kubectl get pods --watch
+   ```
+
+   It may take you up to 20 minutes to deploy all pods, please wait and make sure everything is ready.
+
+   You can tail the logs of the Administration Server with this command:
+
+   ```bash
+   kubectl logs -f domain1-admin-server
+   ```
+
+   The final example of pod output is as following:
+
+   ```bash
+   $ kubectl get pods --watch
+   NAME                                              READY   STATUS      RESTARTS   AGE
+   domain1-admin-server                              1/1     Running     0          11m
+   domain1-create-weblogic-sample-domain-job-4l767   0/1     Completed   0          13m
+   domain1-managed-server1                           1/1     Running     0          3m56s
+   domain1-managed-server2                           1/1     Running     0          3m56s
+   weblogic-operator-56654bcdb7-qww7f                1/1     Running     0          25m
+   ```
+
+   {{% notice tip %}} If Kubernetes advertises the WebLogic pod as `Running` you can be assured the WebLogic Server actually is running because the operator ensures the Kubernetes health checks are actually polling the WebLogic health check mechanism.
```

Comment:
`Running` you -> `Running`, then you (If..., then...)
assured the WebLogic Server actually is running -> assured that WebLogic Server is running
are actually polling -> are polling